utilities/check_mrTrix_Version.m

```diff
@@ -0,0 +1,19 @@
+function mrTrixVersion = check_mrTrix_Version
```

Comment:
Without this call to AFQ_mrtrix_set_ld_path it is impossible to make any
system call to mrtrix. So this means that any function to check if mrtrix
is installed fails.

Jason D. Yeatman, PhD
Assistant Professor, Institute for Learning & Brain Sciences (I-LABS)
Department of Speech & Hearing Sciences
University of Washington
http://brainandeducation.com

On Mon, Jul 11, 2016 at 4:57 PM, Garikoitz Lerma-Usabiaga <
notifications@github.com> wrote:

> In utilities/check_mrTrix_Version.m
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70358150>:
>
> > @@ -0,0 +1,19 @@
> > +function mrTrixVersion = check_mrTrix_Version
>
> Yes, I wrote a note. I don't have a problem with it in OSX, and in two
> different linux boxes (both Red Hat), one normal linux box and the other
> one part of a cluster. Are you using ubuntu? This is the note: % Jason, I
> don't have the AFQ_mrtrix_set_ld_path problem (in linux/OSX), I get rid of
> it with % the matlab call itself. Furthermore, it would be circular, since
> that function % is using the version information as well. We could us a
> function call used in % both versions. Please edit this as it is working
> for you and I % will override it in my version if necessary. I don't
> understand what your problem is, can you send me the error please?
> … <#m_5681299657681076971_>
> On Tue, Jul 12, 2016 at 12:56 AM dstrodtman ***@***.***> wrote: In
> utilities/check_mrTrix_Version.m <#11 (comment)
> <https://github.com/yeatmanlab/AFQ/pull/11#discussion_r70352285>>: > @@
> -0,0 +1,19 @@ > +function mrTrixVersion = check_mrTrix_Version This doesn't
> look like it has been fixed. For this function to run properly we need to
> first call AFQ_mrtrix_set_ld_path. However AFQ_mrtrix_set_ld_path takes as
> an input, the output of the present function. Gari, did this work for you
> on linux? — You are receiving this because you modified the open/close
> state. Reply to this email directly, view it on GitHub <
> https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70352285>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AEWQV5c4XAECeJhde8WXaJsfh1eYoxrYks5qUsoggaJpZM4JGbEU>
> .
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/yeatmanlab/AFQ/pull/11/files/7fc540e2aad16177aeded65d4e6516e6f78cf07e#r70358150>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABsYyh8UDRjPcpSV02PrCA23b81jpIUWks5qUthSgaJpZM4JGbEU>
> .
>